packages/fast-component-explorer/app/explorer.style.ts

```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
     },
     explorer: {},
     explorer_colorPicker: {
-        background: neutralFillRest(() => "#3B3B3B"),
+        background: neutralFillRest(neutralLayerL2),
         border: "none",
         width: "23px",
+        height: "22px",
         ...applyCornerRadius(),
         "&:hover": {
-            background: neutralFillHover(() => "#3B3B3B"),
+            background: neutralFillHover(neutralLayerL2),
         },
         "&:active": {
-            background: neutralFillActive(() => "#3B3B3B"),
+            background: neutralFillActive(neutralLayerL2),
         },
         "&:focus": {
             outline: "none",
         },
     },
-    explorer_viewerRegion: {
+    explorer_devToolsPanel: {
+        boxShadow:
+            "0px 0.6px 1.8px rgba(0, 0, 0, 0.11) inset, 0px 3.2px 7.2px rgba(0, 0, 0, 0.13) inset",
+        width: "100%",
+        overflow: "auto",
+    },
+    explorer_navigationPanel: {
+        overflowY: "auto",
+        overflowX: "hidden",
+        background: neutralLayerL3,
+        boxShadow:
+            "-0.6px 0px 1.8px rgba(0, 0, 0, 0.11) inset, -3.2px 0px 7.2px rgba(0, 0, 0, 0.13) inset",
+    },
+    explorer_paneTitleContainer: {
+        height: "32px",
         display: "flex",
+        boxSizing: "border-box",
+        alignItems: "center",
+        paddingLeft: "10px",
+        borderBottom: format<DesignSystem>("1px solid {0}", neutralDividerRest),
+    },
+    explorer_propertiesPanel: {
+        background: neutralLayerL3,
+        boxShadow:
+            "0.6px 0px 1.8px rgba(0, 0, 0, 0.11) inset, 3.2px 0px 7.2px rgba(0, 0, 0, 0.13) inset",
+    },
+    explorer_toolbar: {
+        padding: "0 10px",
+        height: "32px",
+        display: "flex",
+        verticalAlign: "middle",
```

Comment:
Is vertical align middle doing anything here? With the flex alignment to center everything should be aligned as expected without verticalAlign (typically used w/ inline layouts)

Reply:
yeah I'll remove